packages/client/components/Dashboard.tsx

```diff
@@ -120,7 +108,10 @@ const Dashboard = (props: Props) => {
         )}
         <DashMain>
           <Switch>
-            <Route path='/meetings' component={MeetingsDashRoot} />
+            <Route
+              path='/meetings'
+              render={(routeProps) => <MeetingsDash {...routeProps} viewer={viewer} />}
```

Comment:
not exactly intuitive, but this is how you pass in the viewer

Reply:
I see, I see